exercicios/para-casa/Biblioteca.py

```diff
@@ -0,0 +1,25 @@
+from Livro import Livro
+
+class Biblioteca :
+    def __init__(self):
+        self.livros = [] # propriedade declarada
+
+    def adicionar_livro(self, livro: Livro): # o parametrô livro é do tipo classe Livro
+        # raise NotImplementedError -> não precisamos mais do método de exceção pois agora temos um comportamento:
+        if (not isinstance(livro, Livro)):
+            raise TypeError('mensagem de erro') 
+        self.livros.append(livro)
+
+    def exibir_livro(self, livro: Livro):
+        # raise NotImplementedError
+        self.livros = []
+        print(livro)
+
+    def emprestar_livro(self):
```

Comment:
Aqui está no caminho certo! 

Mas que acha dessa sugestão de algoritmo?

- receber um livro ou nome de um livro como parametro do metodo emprestar_livro
- iterar sobre a lista self.livros (fazer um for)
- verificar se o livro atual do for tem o mesmo nome do parametro recebido
   - Se tiver o mesmo nome então altera o valor do livro de dentro da lista para livro.esta_Emprestado = True (opcionalmente vc pode retornar True também, isso pode facilitar os testes)
   - Se não tiver o mesmo nome então não faz nada (opcionalmente vc pode retornar False também, isso pode facilitar os testes)